packages/reactive-element/src/std-decorators/property.ts

```diff
@@ -112,14 +111,13 @@ export const property = (
           this.requestUpdate(name, oldValue, options);
         },
         init(this: C, v: V): V {
-          if (v !== undefined) {
-            // We need this cast because these decorators are in a separate
-            // compilation unit from ReactiveElement, so can't see the
-            // @internal _$changedProperties field.
-            (
-              this as unknown as {_$changedProperties: PropertyValues}
-            )._$changedProperties.set(name, undefined);
-          }
+          (this.requestUpdate as InternalRequestUpdate)(
```

Comment:
Please add documentation somewhere as per Discord chat:

```
we need to call requestUpdate() for initial values, like we do in the legacy decorators, to both populate the changedProperties map and to perform the initial reflection. But standard decorators do that call too early and it would be an error to read the field to get the previous value at that point
```

Reply:
done!